CHANGELOG.md

```diff
@@ -20,6 +21,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 
+### Changed
+- `PlutusScript` is now aware of which version of Plutus the script is for. The JSON representation has thus changed to reflect this and is not compatible with previously compatible JSON.
+
```

Comment:
Should this be placed under the 2.0.0-alpha section? I'm not sure if the alpha section is preserved in later releases, or if we simply drop the `-alpha` suffix